cpp/ql/lib/ext/std.deque.model .yml

```diff
@@ -0,0 +1,72 @@
+extensions:
```

Comment:
This file is named `cpp/ql/lib/ext/std.deque.model .yml` (spurious space).

Reply:
Fixed in e03f8084e62f087bfed247ad7c9e39b4202fb51d